tests/test_core.py

```diff
@@ -4117,6 +4117,11 @@ def test_mount(self):
     src = open(path_from_root('tests', 'fs', 'test_mount.c'), 'r').read()
     self.do_run(src, 'success', force_c=True)
 
+  def test_getdents64(self):
+    Settings.FORCE_FILESYSTEM = 1
```

Comment:
Actually no idea, copied it from the above test `test_mount`, which has the same, https://github.com/kripken/emscripten/pull/4779/files#diff-cc4345db19ff44863e9122c74e9f383fR4115